src/app/user-page/user-page.component.html

```diff
@@ -0,0 +1,132 @@
+          <mat-card-header>
+            <div mat-card-avatar matBadge="✓" class="mr-2">
+              <img src="../../assets/images/account-logos/github.svg" alt="Github logo" class="w-100" />
+            </div>
+            <mat-card-title class="normal-line-height">GitHub Account</mat-card-title>
+            <mat-card-subtitle class="normal-line-height weight-bold accent-1-dark mt-2 mb-0" fxLayout.lt-md="column" fxLayoutGap="1rem">
+              <span *ngIf="gitHubProfile.name"
+                >Account Name: <span>{{ gitHubProfile?.name }}</span></span
+              >
+              <span>
+                Username:
+                <a href="https://github.com/{{ gitHubProfile?.username }}" target="_blank" class="accent-1-dark">
+                  <span>{{ gitHubProfile?.username }}</span>
+                </a>
+              </span>
+            </mat-card-subtitle>
+          </mat-card-header>
+          <span>
+            <img [src]="gitHubProfile.avatarURL" alt="GitHub avatar" />
+          </span>
+        </div>
+        <div *ngIf="!user?.orcid && !googleProfile" fxLayout="column" fxLayoutAlign="end" class="mt-3">
+          <hr />
+          <span fxLayout fxLayoutAlign="end end">
+            <a href="https://github.com/{{ gitHubProfile?.username }}" target="_blank">
+              <button mat-button class="accent-1-dark-btn small-mat-btn-skin small-btn-structure mt-2">
+                <mat-icon class="mat-icon-sm">open_in_new</mat-icon>
+                View
+              </button>
+            </a>
+          </span>
+        </div>
+      </mat-card>
+      <!-- ORCID -->
+      <mat-card *ngIf="user?.orcid" class="h-100">
+        <mat-card-header>
+          <div mat-card-avatar matBadge="✓" class="mr-2">
+            <img src="../../assets/images/account-logos/orcid.svg" alt="ORCID logo" class="w-100" />
+          </div>
+          <mat-card-title class="normal-line-height">ORCID Account</mat-card-title>
+          <mat-card-subtitle class="accent-1-dark normal-line-height mt-2 mb-0">
+            <a href="https://orcid.org/{{ user?.orcid }}" target="_blank">
+              <span class="weight-bold">https://orcid.org/{{ user?.orcid }}</span>
+            </a>
+          </mat-card-subtitle>
+        </mat-card-header>
+      </mat-card>
+      <!-- Google -->
+      <mat-card *ngIf="googleProfile" fxLayout="column" fxLayoutAlign="space-between" class="h-100">
+        <div fxLayout fxLayoutAlign="space-between">
+          <mat-card-header>
+            <div mat-card-avatar matBadge="✓" class="mr-2">
+              <img src="../../assets/images/account-logos/google.svg" alt="Google logo" class="w-100" />
+            </div>
+            <mat-card-title class="normal-line-height">Google Account</mat-card-title>
+            <mat-card-subtitle class="accent-1-dark normal-line-height weight-bold mt-2 mb-0">
+              <span>Account Name: {{ googleProfile?.name }}</span>
```

Comment:
OK, one more comment :) -- here you use the question mark, but on line 83 you don't.